apps/storefront/src/pages/AccountSetting/index.tsx

```diff
@@ -288,50 +303,96 @@ function AccountSetting() {
 
   return (
     <B3Spin isSpinning={isLoading} background={backgroundColor}>
-      <Box
-        sx={{
-          width: isMobile ? '100%' : '35%',
-          minHeight: isMobile ? '800px' : '300px',
-          '& input, & .MuiFormControl-root .MuiTextField-root, & .MuiSelect-select.MuiSelect-filled, & .MuiTextField-root .MuiInputBase-multiline':
-            {
-              bgcolor: b3HexToRgb('#FFFFFF', 0.87),
-              borderRadius: '4px',
-              borderBottomLeftRadius: '0',
-              borderBottomRightRadius: '0',
-            },
-          '& .MuiButtonBase-root.MuiCheckbox-root:not(.Mui-checked), & .MuiRadio-root:not(.Mui-checked)':
-            {
-              color: b3HexToRgb(getContrastColor(backgroundColor), 0.6),
-            },
-          '& .MuiTypography-root.MuiTypography-body1.MuiFormControlLabel-label, & .MuiFormControl-root .MuiFormLabel-root:not(.Mui-focused)':
-            {
-              color: b3HexToRgb(getContrastColor(backgroundColor), 0.87),
-            },
-          '& .MuiInputLabel-root.MuiInputLabel-formControl:not(.Mui-focused)': {
-            color: b3HexToRgb(getContrastColor('#FFFFFF'), 0.6),
-          },
-        }}
-      >
-        <B3CustomForm
-          formFields={translatedFields}
-          errors={errors}
-          control={control}
-          getValues={getValues}
-          setValue={setValue}
-        />
-
-        <CustomButton
+      <Box>
+        {!isB2BUser && platform === 'catalyst' && (
+          <Box>
+            <Alert
+              severity="info"
+              variant="filled"
+              sx={{
+                width: 'inherit',
+                '& button[title="Close"]': {
+                  display: 'block',
+                },
+                mb: '24px',
+                maxWidth: '1450px',
+
+                '& .MuiAlert-icon': {
+                  padding: '12px 0',
+                },
+
+                '& .MuiAlert-message': {
+                  width: '100%',
+                },
+              }}
+            >
+              <Box display="flex" flexWrap="wrap" justifyContent="space-between" width="100%">
+                <Box>
+                  <Typography variant="subtitle1" fontWeight="800">
+                    {b3Lang('accountSettings.registeredToB2b.title')}
+                  </Typography>
+                  <Typography sx={{ textWrap: 'wrap' }}>
+                    {b3Lang('accountSettings.registeredToB2b.description')}
+                  </Typography>
+                </Box>
+                <Typography
+                  component={Link}
+                  to="/registeredbctob2b"
+                  sx={{ textDecoration: 'none', textTransform: 'uppercase' }}
+                  fontWeight="bold"
+                  color="#fff"
+                >
+                  {b3Lang('accountSettings.registeredToB2b.upgrade')}
+                </Typography>
+              </Box>
+            </Alert>
+          </Box>
```

Comment:
We might want to extract this to its own component. There's nothing here that we have to share with the parent. This helps reduce context-overload.


```tsx
{!isB2BUser && platform === 'catalyst' && <UpgradeBanner />}
```